packages/@aws-cdk/aws-apprunner/lib/service.ts

```diff
@@ -0,0 +1,722 @@
+import * as ecr from '@aws-cdk/aws-ecr';
+import { DockerImageAsset } from '@aws-cdk/aws-ecr-assets';
+import * as iam from '@aws-cdk/aws-iam';
+import * as cdk from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import { CfnService } from './';
+
+/**
+ * The image repository types
+ */
+export enum ImageRepositoryType {
+  /**
+   * Amazon ECR Public
+   */
+  ECR_PUBLIC = 'ECR_PUBLIC',
+
+  /**
+   * Amazon ECR
+   */
+  ECR = 'ECR',
+}
+
+/**
+ * The number of CPU units reserved for each instance of your App Runner service.
+ */
+export class Cpu {
+  /**
+   * 1 vCPU
+   */
+  public static readonly ONE_VCPU = Cpu.of('1 vCPU')
+
+  /**
+   * 2 vCPU
+   */
+  public static readonly TWO_VCPU = Cpu.of('2 vCPU')
+
+  /**
+   * Custom CPU unit
+   *
+   * @param unit custom CPU unit
+   */
+  public static of(unit: string) { return new Cpu(unit); }
```

Comment:
Is there a documentation page somewhere which tells you what values are valid? If so, would be great to link it there (even if the only options _at this time_ are those for which you have constants above...).

Reply:
Unfortunately we have no document telling all valid values. From the console UI the only available values are `1 vCPU` and `2 vCPU`. I am adding a link to the cloudformation document here as a reference.